l1-contracts/contracts/state-transition/StateTransitionManager.sol

```diff
@@ -183,7 +183,7 @@ contract StateTransitionManager is IStateTransitionManager, ReentrancyGuard, Own
             txType: SYSTEM_UPGRADE_L2_TX_TYPE,
             from: uint256(uint160(L2_FORCE_DEPLOYER_ADDR)),
             to: uint256(uint160(L2_SYSTEM_CONTEXT_SYSTEM_CONTRACT_ADDR)),
-            gasLimit: $(PRIORITY_TX_MAX_GAS_LIMIT),
+            gasLimit: 72_000_000,
```

Comment:
Would it be better to move it to config? and use as a named value?

Reply:
agreed, in the long term we may try to use some foundry tools so consolidating into a config file would make that easier

Reply:
@koloz193 Indeed place this constant in `Config.sol`

Reply:
Fixed in b356ac28066542316594174547204ff16d950397